### DISCUSSION TITLE
Allow lock the list of properties

### DISCUSSION THREAD

**Discussion by CodeLazier** [godotengine/godot-proposals#5000] [Editor]
I have to drag the texture and then sprite into the texture property. for a single sprite, this often requires many tests.
I wish there was a menu option that locked or Pin the list of properties of nodes so that it would not be easy to switch.
Considering that when selecting other nodes, the user is mistaken for the property list of the currently selected node because it is locked, you can use a flashing border to remind the property list that it is currently locked.

**Comment by Calinou** [comment]
In 3.5 and later, drag-and-dropping from the FileSystem dock won't switch nodes until you release the mouse button.
